<launch>

  <param name="use_sim_time" value="false" />

  <!-- Set the name of the map yaml file: can be overridden on the command line. -->
  <arg name="map" default="test_map.yaml" />
  <node name="map_server" pkg="map_server" type="map_server" args="$(find bitathome_navigation_control)/maps/big.yaml" />

  <include file = "$(find bitathome_navigation_control)/launch/navigation_configuration.launch" />

  <node pkg="laser_scan_matcher" type="laser_scan_matcher_node" name="laser_scan_matcher_node" output="screen">
    <param name="fixed_frame" value="odom" />
  </node>

  <node pkg="bitathome_hardware_control" name="hc_motor_cmd" type="hc_motor_cmd.py" output="screen" />
  <include file="$(find bitathome_navigation_control)/launch/tb_amcl.launch" />
  <include file="$(find bitathome_navigation_control)/launch/shopping_move_base.launch" />
  <node pkg="bitathome_navigation_control" name="nc_move_base_server" type="nc_move_base_server.py" output="screen"/>
  <node pkg="bitathome_move_control" name="mc_move_base_control" type="mc_move_base_control.py" />

  <node pkg="tf" type="static_transform_publisher" name="map_odom_broadcaster" args="0 0 0 0 0 0 /map /odom 100" />
  <node name="soundplay_node" pkg="sound_play" type="soundplay_node.py" output="screen" />
  <node name="AudioPlay" pkg="bitathome_remote_control" type="AudioPlay.py" output="screen" />

</launch>
